_dprhtml/js/format.js

```diff
@@ -458,6 +458,7 @@ function convtitle(nikaya,book,una,vna,wna,xna,yna,zna,hiert,oneline,click)
     }
 
     var onet = '<b style="color:'+DPR_G.DPR_prefs[col[w++]]+'">' + translit(toUni(thisname)) + '</b>';
+    $('#dialog-bookmarkInput').val(translit(toUni(thisname)));
```

Comment:
there is ko binding already on dialog-bookmarkInput - lets use that and not jquery

Reply:
done